docs/hotfix-procedure.md

```diff
@@ -0,0 +1,25 @@
+# Hotfix procedure 
+
+The first step is to branch off of main and propose a patch version.
+
+1. `git checkout main`
+2. `git pull origin main`
+3. `git checkout -b your-branch-name`
+4. Bump the patch version in `package.json`
+
+Next you need to fix the bug. Unfortunately this guide can't tell you how to fix the bug. After that though:
+
+5. Create a pull request to merge your changes into the `main` branch.
+6. Two code reviews are required for hotfixes. Reviewers, be careful, you will probably be the final set of eyes before this goes to production.
+7. Merge the pull request 
+8. Create a new tag for your release and deploy to production. 
+
+Finally, you need to get your change into the develop branch as well.
+
+9. `git pull origin develop`
```

Comment:
Wait.... we shouldn't be pulling FROM develop onto main or a main-adjacent branch.

Instead, I propose here:
1. git checkout develop
2. git pull origin develop (good habit)
3. git checkout -b new-develop-offshoot
4. git merge nameOfBranchFromStep3Section1
5. Resolve merge conflicts.
6. Push
7. Issue PR into develop.

Reply:
I think this is just a shortcut for those steps? If you are on your hotfix branch you will be one commit ahead of main and many commits behind develop. So when you pull develop you will initiate the merge of develop into your hotfix

Reply:
Ah ok. That's fine, then. Just gives me the heebie-jeebies.

Reply:
I'm gonna rewrite it a little to make it very clear what branch you should be on when you run these commands